adanet-co2-monitor.ino

```diff
@@ -51,6 +54,8 @@ static Adafruit_DPS310 dps;
 static uint32_t error;
 static char message[MESSAGE_SIZE];
 static Preferences pref;
+static constexpr uint8_t CO2_VAL_STRING_LEN = 20;
```

Comment:
Nit: keep `constexpr`s together up top

Reply:
Also, is the max size really 20?

Reply:
Originally there because it is used in the next line but ok lets move it up

re length, I think it is at most 5 char + terminator so 6. I reduced it to 10.

Reply:
Gotcha, was mostly just curious (since memory isn't a concern)